packages/mux-player/test/web-test-runner.config.mjs

```diff
@@ -19,7 +19,7 @@ export default {
     esbuildPlugin({
       ts: true,
       json: true,
-      loaders: { '.css': 'text', '.svg': 'text' },
+      loaders: { '.css': 'text', '.svg': 'text', '.html': 'text' },
```

Comment:
suggestion(non-blocking): Since this isn't "standard html," (aka it is intended to be used as a template) and per some of the other build-level callouts (css & svg inclusions), we should consider making this extension something like `.html.templ` (or whatever feels right that isn't simply `.html`) so we can distinctly target it for build tools going forward.

Reply:
I don't think that's needed here, it's valid HTML. it being the html extension right away gets the right syntax in the editor. if we had many html files we import maybe but this is really a rare case

Reply:
right, this was related to being able to externalize e.g. the svg files and the css file